lua/starfall/sflib.lua

```diff
@@ -86,6 +86,10 @@ hook.Add("InitPostEntity","SF_SanitizeTypeMetatables",function()
 			return unpack(result)
 		end
 	end
+	
+	if SERVER and util.NetworkStringToID("sf_moneyrequest") ~= 0 then
```

Comment:
I'd prefer if this wasn't in sflib.lua. If this and the old version have the same file path, shouldn't there be no issue? One will just take priority over the other.